internal/instrumented_resp_connection/instrumented_resp_connection.go

```diff
@@ -6,10 +6,10 @@ import (
 
 	resp_connection "github.com/codecrafters-io/redis-tester/internal/resp/connection"
 	resp_value "github.com/codecrafters-io/redis-tester/internal/resp/value"
-	"github.com/codecrafters-io/tester-utils/test_case_harness"
+	"github.com/codecrafters-io/tester-utils/logger"
 )
 
-func defaultCallbacks(stageHarness *test_case_harness.TestCaseHarness, logPrefix string) resp_connection.RespConnectionCallbacks {
+func defaultCallbacks(Logger *logger.Logger, logPrefix string) resp_connection.RespConnectionCallbacks {
```

Comment:
```suggestion
func defaultCallbacks(logger *logger.Logger, logPrefix string) resp_connection.RespConnectionCallbacks {
```

(havent' seen us naming variables with capitals elsewhere)

Reply:
Oh, is this to avoid conflicts with the package? In that case, I'd just import the package as `tester_utils_logger` or something like that. With this naming scheme it isn't clear why the variable is named differently.